scripts/sct_download_data.py

```diff
@@ -70,9 +70,8 @@ def main(args=None):
             "https://www.neuro.polymtl.ca/_media/downloads/sct/20180525_sct_example_data.zip",
         ],
         "sct_testing_data": [
-            "https://github.com/sct-data/sct_testing_data/releases/download/r20200707232231/sct_testing_data-r20200707232231.zip",
-            "https://osf.io/download/5f053c176cbca300dad363d3/"
-        ],
+            "https://github.com/sct-data/sct_testing_data/releases/download/r20200504-13-g713fcb6/sct_testing_data-r20200504-13-g713fcb6.zip",
+            "https://osf.io/download/5f516b634f1e5e00226f0599/"],
```

Comment:
I'm a bit confused about the naming convention here. Why have we gone from July 7th to May 4th ([`r20200504`](https://github.com/sct-data/sct_testing_data/releases/tag/r20200504-13-g713fcb6)) in this newest dataset release?

Reply:
wow! good catch, i didn't notice the date. however, i was not able to specify the folder name and i had no time to dig. i opened an issue [here](https://github.com/neuropoly/distriploy/issues/9); if it is fixed quickly (ie within the next hours), we can re-create a tag with proper naming before merging this PR.

Reply:
ok, i manually edited the zip file on GH and OSF and updated the URL: 4864a95ecdcdcbdb753dcf73216205f21f7e9c92